e2e/commands/extentsion.e2e.js

```diff
@@ -0,0 +1,57 @@
+import chai, { expect } from 'chai';
+import Helper from '../e2e-helper';
+
+const assertArrays = require('chai-arrays');
+
+chai.use(assertArrays);
+
+describe('bit extension system', function () {
+  this.timeout(0);
+  const helper = new Helper();
+  after(() => {
+    helper.destroyEnv();
+  });
+  beforeEach(() => {
+    helper.reInitLocalScope();
+  });
+
+  it.skip('Exception from hook should not affect others which are registers for that hook', () => {});
+  it.skip('Should be able to trigger a hook', () => {});
+  describe('new command for extension', () => {
+    it.skip('Should be shown in --help', () => {});
+    it.skip('Should be able to run "bit <command> -h"', () => {});
+    it.skip('Should trigger loader', () => {});
+  });
+  describe('extension can register a new hook', () => {
+    it.skip('Should be able to trigger the hook they created', () => {});
+    it.skip('Should not be able to trigger a hook they didnt create', () => {});
+  });
+  it.skip('2 different extentions create hook with the same name', () => {});
+  describe('extension logger', () => {
+    it.skip('Should write to log', () => {});
+    it.skip('Should check that the name of the extension appears in the log', () => {});
+  });
+  describe('default hooks', () => {
+    it.skip('default hook should run when triggered', () => {});
+  });
+  describe('when importing another version of the same extension', () => {
```

Comment:
not all cases inside are about another version
change to importing extension
add this describe (with another version) inside 
put the 
Should have the latest import in the bit.json
Should copy the config / options in bit.json to the new version 
inside the with another version
and the other 2 in the importing extension